Installation/cmake/modules/CGAL_Macros.cmake

```diff
@@ -287,7 +287,9 @@ if( NOT CGAL_MACROS_FILE_INCLUDED )
 
         if (OPENGL_FOUND)
           add_definitions("${OPENGL_DEFINITIONS}")
-          include_directories( SYSTEM "${OPENGL_INCLUDE_DIR}" )
+          if ( NOT "${OPENGL_INCLUDE_DIR}" STREQUAL "" )
```

Comment:
As said in my commit 561bd80d961e8c9178aea6bde59317061f9efed9: on Windows, OPENGL_INCLUDE_DIR is empty even when OpenGL is found. Probably because it's already in the MSVC paths.